src/styles/_quote.scss

```diff
@@ -0,0 +1,14 @@
+.quote {
+  text-align: center;
+  color: #078d7c;
+  font-style: italic;
+  margin: 170px 0 100px;
+
+  @media (max-width: grav-breakpoint(large)) {
+    margin: 120px 0 70px;
+  }
+
+  @media (max-width: grav-breakpoint(medium)) {
+    margin: 70px 0 50px;
+  }
+}
```

Comment:
Needs replacing with Gravity vars

Reply:
What _needs replacing with Gravity vars_ ?

Reply:
I was mainly talking about the `color` but possibly the margins (multiples of $grav-sp-xl maybe). Not a biggie for now though.